# Metagenomics-anlaysis: Metagenomics analysis of human lung data
> Objective: Analyze lungData file and run script with RMarkdown. By utilizing metagenomicSeq tool, we can learn about the abundance and correlation between two trial datasets.

## Author
Ge Zhu < zhu.ge@husky.neu.edu>
Date: 11-04-2019 

Tools:
- RStudio
- BioConductor
- MetagenomeSeq

## Data Description: 
Human lung microbiome [1]: The lung microbiome consists of respiratory ora sampled from six healthy individuals. Three healthy nonsmokers and three healthy smokers. The upper lung tracts were sampled by oral wash and oro-/nasopharyngeal swabs. Samples were taken using two bronchoscopes, serial bronchoalveolar lavage and lower airway protected brushes.

Humanized gnotobiotic mouse gut [2]: Twelve germ-free adult male C57BL/6J mice were fed a low-fat, plant polysaccharide-rich diet. Each mouse was gavaged with healthy adult human fecal material. Following the fecal transplant, mice remained on the low-fat, plant polysacchaaride-rich diet for four weeks, following which a subset of 6 were switched to a high-fat and high-sugar diet for eight weeks. Fecal samples for each mouse went through PCR amplication of the bacterial 16S rRNA gene V2 region weekly.

## Reference
Sohn, B., M., & Lingling. (2015, March 19). robust approach for identifying differentially abundant features in metagenomic samples. Retrieved from https://academic.oup.com/bioinformatics/article/31/14/2269/256302 
